OZprivate/rawJS/OZTreeModule/src/factory/midnode.js

```diff
@@ -412,10 +412,11 @@ class Midnode {
       num_threatened += this.get_attribute("iucnEN");
       num_threatened += this.get_attribute("iucnCR");
     }
+    let num_extant = this.richness_val - (this.get_attribute("iucnEX") + this.get_attribute("iucnEW"));
```

Comment:
Why not add these to ``num_threatened`` as with the other values, rather than subtracting from ``richness_val``? I don't think that ``num_extant`` does what you want:

```
    num_threatened > num_extant * 0.5
==> iucnVU + iucnEN + iucnCR > richness * 0.5 - (iucnEX + iucnEW) * 0.5
==> iucnVU + iucnEN + iucnCR + (iucnEX + iucnEW) * 0.5 > richness * 0.5
==> iucnVU + iucnEN + iucnCR + iucnEX * 0.5 + iucnEW * 0.5 > richness * 0.5
```

Reply:
I'm going with the definition here under Categories:

https://en.m.wikipedia.org/wiki/IUCN_Red_List

I read your expansion but I think it does what we want. It's confusing to expand because the extinct categories are actually adding to richness already, so this removes them from the calculation.